.github/workflows/vm-nightly-test.yml

```diff
@@ -3,64 +3,64 @@ name: VM Nightly Test
-      - run: 'npm run test:API'
-        env:
-          CI: true
-        working-directory: '${{ env.cwd }}'
-      - run: 'npm run test:API:browser'
-        env:
-          CI: true
-        working-directory: '${{ env.cwd }}'
-    env:
-      cwd: '${{github.workspace}}/packages/vm'
-  test-state:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/setup-node@v1
-        with:
-          node-version: 12.x
-      - uses: actions/checkout@v1
-      - run: npm install
-        working-directory: '${{ env.cwd }}'
-      - run: 'npm run test:state:allForks'
-        env:
-          CI: true
-        working-directory: '${{ env.cwd }}'
-    env:
-      cwd: '${{github.workspace}}/packages/vm'
-  test-blockchain:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/setup-node@v1
-        with:
-          node-version: 12.x
-      - uses: actions/checkout@v1
-      - run: npm install
-        working-directory: '${{ env.cwd }}'
-      - run: 'npm run test:blockchain'
-        env:
-          CI: true
-        working-directory: '${{ env.cwd }}'
-    env:
-      cwd: '${{github.workspace}}/packages/vm'
+  # test-api:
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - uses: actions/setup-node@v1
+  #       with:
+  #         node-version: 12.x
+  #     - uses: actions/checkout@v1
+  #     - run: npx lerna bootstrap --scope ethereumjs-vm --ignore-scripts --include-dependencies --no-ci
+  #     - run: npx lerna run --scope ethereumjs-vm --include-dependencies build
+  #     - run: 'npm run test:API'
+  #       env:
+  #         CI: true
+  #       working-directory: '${{ env.cwd }}'
+  #     - run: 'npm run test:API:browser'
+  #       env:
+  #         CI: true
+  #       working-directory: '${{ env.cwd }}'
+  #   env:
+  #     cwd: '${{github.workspace}}/packages/vm'
+  # test-state:
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - uses: actions/setup-node@v1
+  #       with:
+  #         node-version: 12.x
+  #     - uses: actions/checkout@v1
+  #     - run: npx lerna bootstrap --scope ethereumjs-vm --ignore-scripts --include-dependencies --no-ci
+  #     - run: npx lerna run --scope ethereumjs-vm --include-dependencies build
+  #     - run: 'npm run test:state:allForks'
+  #       env:
+  #         CI: true
+  #       working-directory: '${{ env.cwd }}'
+  #   env:
+  #     cwd: '${{github.workspace}}/packages/vm'
+  # test-blockchain:
+    # runs-on: ubuntu-latest
+    # steps:
+    #   - uses: actions/setup-node@v1
+    #     with:
+    #       node-version: 12.x
+    #   - run: npx lerna bootstrap --scope ethereumjs-vm --ignore-scripts --include-dependencies --no-ci
+    #   - run: npx lerna run --scope ethereumjs-vm --include-dependencies build
+    #     working-directory: '${{ env.cwd }}'
+    #   - run: 'npm run test:blockchain'
+    #     env:
+    #       CI: true
+    #     working-directory: '${{ env.cwd }}'
+    # env:
+    #   cwd: '${{github.workspace}}/packages/vm'
```

Comment:
After migrating to a monorepo setup, these cache warm-up jobs are quite useless, as they only reach the shallow `node_modules`. We should reactivate them later, probably along with Lerna package hoisting.